src/cgame/cg_gameinfo.cpp

```diff
@@ -41,11 +41,11 @@ CG_ParseInfos
 int CG_ParseInfos( const char *buf, int max, char *infos[] )
 {
 	char *token;
-	int  count;
+	int  count = 0;
 	char key[ MAX_TOKEN_CHARS ];
 	char info[ MAX_INFO_STRING ];
 
-	count = 0;
+	int infoPostfixLen = strlen( "\\num\\" ) + strlen( va( "%d", MAX_ARENAS ) );
```

Comment:
Since strlen returns a size_t, it's very likely that size_t + int would return a size_t anyway.
But yeah, I guess you can change BG_Alloc to accept a size_t.